src/controllers/management-controller.ts

```diff
@@ -68,23 +68,23 @@ export class ManagementController {
   ): Promise<IdResponse> {
     return this.#inner
       .request(context.management, {
-        path: "/v2/assets",
+        path: "/v3/assets",
         method: "POST",
         apiToken: context.apiToken,
         body: {
           ...input,
           "@context": this.defaultContextValues,
         },
       })
-      .then(body => expand(body, () => new IdResponse()));
+      .then((body) => expand(body, () => new IdResponse()));
```

Comment:
just a question, why do we put brackets also when there's only a parameter?

Reply:
This is done automatically by my linter